Assets/MixedRealityToolkit/MixedReality.Toolkit.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.MixedReality.Toolkit</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft,MixedReality</owners>
+    <licenseUrl>https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/License.txt</licenseUrl>
+    <projectUrl>https://github.com/Microsoft/MixedRealityToolkit-Unity</projectUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>The Mixed Reality Toolkit is a collection of scripts and components intended to accelerate development of applications targeting Microsoft HoloLens and Windows Mixed Reality headsets.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <releaseNotes>$releaseNotes$</releaseNotes>
+    <tags>Unity MixedReality</tags>
+  </metadata>
+  <files>
+        <file src="**" target="" exclude="Sharing\**;Sharing.meta" />
```

Comment:
we can probably remove this as there is nothing of consequence here to filter out

Reply:
Ah, I see. We ran into an issue in the past where RakNet was not passing customer rules around OSS for leveraging the MRTK so we've been building without it locally for a while now. Is RakNet coming back in? What's the plan for the Sharing stuff?

Reply:
We're not sure yet. Let's add it in when we cross that bridge.